pkg/descheduler/evictions/evictions.go

```diff
@@ -57,6 +55,8 @@ type PodEvictor struct {
 	nodepodCount               nodePodEvictedCount
 	namespacePodCount          namespacePodEvictCount
 	metricsEnabled             bool
+	eventBroadcaster           events.EventBroadcasterAdapter
```

Comment:
`eventBroadcaster` is not used anywhere in the code, the parameter can be dropped